### PR TITLE
Job equipment

### PR DESCRIPTION
**Pull request by Loganbacca**
I decided to get something done that really needs to be done for the server to be game ready.

 - Fixed job preference selection
 - Added several utility belt variants (for engineer starting equipment)
 - Removed assistant job selection code
 - Swapped citizen and liason bit flags so citizen is the default "no preferences available" job
 - Fixed character creation not adding items to backpacks
 - Added food and water to survival box (replaces air tank and mask)
 - Added box of diving equipment (medium)
 - Gave Marshal an energy gun as part of starting equipment

If you've got any suggestions for what equipment roles should or shouldn't have, please comment here. This isn't a personal project, so I won't be offended if you think something is totally wrong and needs to change, this is a getting things done PR.

**Current gear**
(all roles have basic randomly selected clothing, food, and water unless otherwise stated)

Visitor:
 - no survival supplies

Doctor:
 - labcoat

Janitor:
 - galoshes
 - purple hat

Gardener:
 - leather gloves
 - apron
 - plant analyzer

Civil engineer:
 - hard hat
 - diving gear
 - utility belt with: screwdriver, wrench, crowbar, welding tool, wirecutters, flashlight, engineering tape

Electrician:
 - insulated gloves
 - box of lightbulbs
 - utility belt with: screwdriver, wirecutters, t-ray scanner, multitool, crowbar, 2x cable coils

Emergency services:
 - red hard hat
 - plasma cutter
 - diving gear
 - utility belt with: red crowbar, mini extinguisher, flashlight, flare, engineering tape

Miner:
 - crowbar
 - ore bag

Scientist (including xenobiologist and field technician):
 - labcoat

Petty officer:
 - (no random gear)
 - uniform (including hat)
 - jackboots
 - security pda
 - grey gloves
 - sunglasses (underwater!)

### CODE REVIEW

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -61,8 +61,36 @@
 	new /obj/item/weapon/weldingtool(src)
 	new /obj/item/weapon/crowbar(src)
 	new /obj/item/weapon/wirecutters(src)
-	new /obj/item/stack/cable_coil(src,30,pick("red","yellow","orange"))
+	new /obj/item/stack/cable_coil(src,30,pick("red","yellow","green","blue","pink","orange","cyan","white"))
```
Comment (PsiOmegaDelta) [inline code/game/objects/items/weapons/storage/belt.dm]: If memory serves, should you not supply any color to coils at all they'll select a valid random color for you.

**Comment by Zuhayr** [comment]
:+1: Thank you for doing this.

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (Loganbacca) [inline code/game/objects/items/weapons/storage/belt.dm]: Good point, thanks.

**Comment by Loganbacca** [comment]
No problem bro. 

Are the gear suggestions here something you still want in? http://wetskrell.org/water/viewtopic.php?p=39#p39
i.e.

**Comment by Zuhayr** [comment]
Yeah, that'd be good. Some of these may need balancing down the track though, like starting with a pressure suit.

**Comment by Loganbacca** [comment]
Any other specific gear anyone would like to see added or removed?

If not feel free to merge this.

**Pull request merged**